docs/buildspecs/buildspec_overview.rst

```diff
@@ -447,6 +448,28 @@ Let's build the test and see the output.
 
    .. command-output:: buildtest build -b tutorials/test_status/file_and_dir_check.yml
 
+.. _symlink_check:
+
+Passing test based on Symbolic Link
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+buildtest can configure PASS/FAIL of test based on the status of symbolic link. This can be useful if your test will create a symbolic link to a file or directory and 
+will pass if the symbolic link is not broken.  
+
+You can use the ``is_symlink`` property which expects a list of values that are checked for symbolic links.
+In the example below, the test will pass as all the values are valid symbolic links and are not broken. Note that variable and shell expansion
```

Comment:
Either add a new line to start a new paragraph or remove the new line. Right now it looks like a new line is added in middle of a paragraph

Reply:
@shahzebsiddiqui Okay, I have changed this.